Main.java

```diff
@@ -4,26 +4,42 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    static final String FILE_TYPE_TXT = ".txt";
+    static final String FILE_TYPE_JSON = ".json";
+    static final String REMOVE_SPECIAL_CHAR = "\\p{Punct}";

```

Comment:
How is REMOVE_SPECIAL_CHAR and EXCLUDE_SPECIAL_CHAR different?

Reply:
They are the same actually. EXCLUDE_SPECIAL_CHAR is not used anywhere in the program. I'll remove it.